<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">

   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>输入路径</string>
    </property>
   </widget>

   <widget class="QLineEdit" name="input_line">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>

   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>输出路径</string>
    </property>
   </widget>

   <widget class="QLineEdit" name="output_line">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>

   <widget class="QPlainTextEdit" name="main_text">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
   </widget>


   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>excel名称</string>
    </property>
   </widget>


   <widget class="QPushButton" name="excel2xls_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>95</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>拆分excel</string>
    </property>
   </widget>


   <widget class="QPushButton" name="xls2excel_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>95</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>合并excel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>

   <widget class="QMenu" name="menuexcel">
    <property name="title">
     <string>excel小工具</string>
    </property>
   </widget>
   <addaction name="menuexcel"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
